Parser/asdl_c.py

```diff
@@ -679,15 +688,40 @@ def visitModule(self, mod):
     return 0;
 }
 
+static inline PyObject *
+find_field_default(PyObject *field_default)
+{
+    PyObject *ret = NULL;
+    if (PyUnicode_GET_LENGTH(field_default) < 1) {
+        goto result;
+    }
+    switch (PyUnicode_READ_CHAR(field_default, 0)){
+    case '?':
+        ret = Py_None;
+        goto result;
+    case '*':
+        ret = PyList_New(0);
+        goto result;
+    default:
+        goto result;
+    }
+  result:
+    Py_DECREF(field_default);
```

Comment:
Why are you decrementing the reference of `field_default` here? You are already decrementing it in the caller, no?

Reply:
no